src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx

```diff
@@ -10,18 +10,51 @@ import {
   Spacer,
 } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
 import { FairHeader_fair } from "v2/__generated__/FairHeader_fair.graphql"
 import { FairTimingFragmentContainer as FairTiming } from "./FairTiming"
-import { FairHeaderImageFragmentContainer as FairHeaderImage } from "./FairHeaderImage"
 import { FairHeaderIconFragmentContainer } from "./FairHeaderIcon"
+import {
+  FairEditorialFragmentContainer,
+  FAIR_EDITORIAL_AMOUNT,
+} from "../FairEditorial"
+import { FairCollectionsFragmentContainer } from "../FairCollections"
+import { FairFollowedArtistsFragmentContainer } from "../FairFollowedArtists"
+import { useSystemContext } from "v2/System"
 import { RouterLink } from "v2/System/Router/RouterLink"
 
+const TextWithNewlines = styled(Text)`
+  white-space: pre-wrap;
+`
+
+interface FairInfoSectionProps {
+  label?: string
+  info?: string
+  isHTML?: boolean
+}
+
+const FairInfoSection: React.FC<FairInfoSectionProps> = ({
+  label,
+  info,
+  isHTML = false,
+}) => (
+  <>
+    {label && <Text variant="md">{label}</Text>}
+    {isHTML
+      ? info && <HTML variant="md" html={info} />
+      : info && <TextWithNewlines variant="md">{info}</TextWithNewlines>}
+
```

Comment:
You can use such code, because [you check](https://github.com/artsy/force/blob/e7a7b3ff11b03d8c484771dcc3240b3fd8cb577e/src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx#L90) the `info` prop before rendering the `FairInfoSection` component

```javascript
{isHTML ? <HTML variant="md" html={info} /> : <TextWithNewlines variant="md">{info}</TextWithNewlines>}
```